proof/refine/ARM/Ipc_R.thy

```diff
@@ -5653,21 +5654,21 @@ lemma replyRemoveTCB_sym_heap_scReplies [wp]:
          apply (clarsimp simp: isHead_to_head)
          apply (intro conjI; intro allI impI)
           apply clarsimp
-          apply (subgoal_tac "replyPrevs_of s r = Some ya")
+          apply (subgoal_tac "replyPrevs_of s rv = Some ya")
            apply (rule replyNexts_Some_replySCs_None)
            apply (simp add: sym_refs_replyNext_replyPrev_sym[symmetric])
           apply (clarsimp simp: obj_at'_real_def ko_wp_at'_def projectKOs opt_map_def)
          apply (intro conjI; intro allI impI)
           apply clarsimp
           apply (drule ko_at'_inj, assumption, simp)
           apply (clarsimp simp: isHead_to_head)
-          apply (subgoal_tac "replyPrevs_of s r = Some rPtr")
+          apply (subgoal_tac "replyPrevs_of s rv = Some rPtr")
            apply (insert replyNexts_Some_replySCs_None)
            apply (simp add: sym_refs_replyNext_replyPrev_sym[symmetric])
            apply (clarsimp simp: obj_at'_real_def ko_wp_at'_def projectKOs opt_map_def)
           apply (clarsimp simp: obj_at'_real_def ko_wp_at'_def projectKOs opt_map_def)
          apply clarsimp
-         apply (subgoal_tac "replyPrevs_of s r = Some ya")
+         apply (subgoal_tac "replyPrevs_of s rv = Some ya")
```

Comment:
Same `rename_tac` situation as above (potentially these could be the same rename_tac, hard to see in the diff)